Gemfile

```diff
@@ -5,7 +5,6 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in nepali_calendar.gemspec
 gemspec
 
-group :test do
-  gem 'byebug'
-  gem 'pry'
-end
+# group :test do
+#   gem 'pry'
+# end
```

Comment:
lets remove these comments..

these dependencies should come from gemspec as development dependency instead

eg:
Gem::Specification.new do |spec|
  # Other gemspec configurations

  spec.add_development_dependency "byebug"
end